cypress/e2e/home-page.cy.ts

```diff
@@ -17,8 +13,10 @@ describe('Home page behavior', () => {
       cy.get('main');
       cy.get('footer');
       // No console errors are produced.
-      cy.wait(DELAY_MS).then(() => {
-        expect(windowErrorSpy).to.not.be.called;
+      cy.window().then((win) => {
+        cy.wait(DELAY_MS).then(() => {
+          expect(cy.spy(win.console, 'error')).to.not.be.called;
```

Comment:
This won't work. You need to set up the spy before the test runs. See signpost-template for how it's done.